metricbeat/modules.d/system.yml

```diff
@@ -23,3 +23,7 @@
   processors:
   - drop_event.when.regexp:
       system.filesystem.mount_point: '^/(sys|cgroup|proc|dev|etc|host|lib)($|/)'
+
+- module: system
+  period: 15m
+  metricsets: [uptime]
```

Comment:
I'm wondering if this format (instead of a list with `-`) may confuse some users, as we don't follow the same pattern in the whole file. WDYT?